test/integration/armadaserver_controller_test.go

```diff
@@ -36,29 +35,31 @@ var _ = Describe("Armada Operator", func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(string(stdoutBytes)).To(Equal("armadaserver.install.armadaproject.io/armadaserver-e2e created\n"))
 
-				time.Sleep(2 * time.Second)
-
 				as := installv1alpha1.ArmadaServer{}
 				asKey := kclient.ObjectKey{Namespace: "default", Name: "armadaserver-e2e"}
-				err = k8sClient.Get(ctx, asKey, &as)
-				Expect(err).NotTo(HaveOccurred())
+				Eventually(func() error {
+					return k8sClient.Get(ctx, asKey, &as)
+				}, "2s", "10ms").ShouldNot(HaveOccurred())
```

Comment:
Could we refactor that as a constant so it would be easy to update these in one place?

Reply:
okay will do